### PR TITLE
doc(distr): add CP section and Vesting account note

### PR DESCRIPTION
**Pull request by tbruyelle**
[Rendered version](https://github.com/tbruyelle/govgen-proposals/blob/tbruyelle/doc/distr-update/001_ATONE_DISTRIBUTION.md)

### CODE REVIEW

**Review by adam-hanna — APPROVED** [review]

File: 001_ATONE_DISTRIBUTION.md [hunk 1]
```diff
@@ -164,8 +169,22 @@ ATONE distribution will be of approximately ~97 Millions.
...
+The same amount will be minted to feed a reserved address (say
+`0x00000000000000000builddao`) for future funding of the Treasury DAO.
+
+- The proposed ATONE supply is:
+  ```math
+  ATONE Distributed + Community Pool + Reserved address
+  96,997,800        + 5,388,766      + 5,388,766        = 107,775,332
+  ```
+  which is around ~3/10 (31.43%) of the ATOM supply at the time of proposal 848
+  (342,834,268).
```
Comment (jaekwon) [inline 001_ATONE_DISTRIBUTION.md:187]: [ ] i don't think we need to add this past sentence in here, but we could add it elsewhere on top or bottom as summary? it's just in the wrong section. 

[ ] where the ICF is mentioned as being slashed, we should refer to this section.

Comment (tbruyelle) [reply 001_ATONE_DISTRIBUTION.md:192]: @jaekwon I have addressed both here c6aaa0475364bf17a8bece6d6cf6d17e7c157529

**Review by zivkovicmilos — APPROVED** [review]
Numbers check out to me when following the guide for generating them 💯 
The guides were pretty good, thank you @tbruyelle.

I've also went through parts of the `govbox` code that relate to distribution, and it also seems alright 👍 

I've found a few broken links, but nothing major:
- block 18010657 export link not working https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json
- block 18010658 export link not working https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010658.json

**Comment by tbruyelle** [comment]
> Numbers check out to me when following the guide for generating them 💯 The guides were pretty good, thank you @tbruyelle.
> 
> I've also went through parts of the `govbox` code that relate to distribution, and it also seems alright 👍
> 
> I've found a few broken links, but nothing major:
> 
> * block 18010657 export link not working [atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json](https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json)
> * block 18010658 export link not working [atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010658.json](https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010658.json)

Thank you for the fast verification @zivkovicmilos !
For the audience, the broken links are in the govbox documentation, they are [fixed now](https://github.com/atomone-hub/govbox/commit/3468d08cb64d300c160f7837c5aff20a457bf3cc), it was a typo in the bucket directory. Thanks for pointing them!

File: 001_ATONE_DISTRIBUTION.md [hunk 2]
```diff
@@ -164,8 +175,24 @@ ATONE distribution will be of approximately ~97 Millions.
...
-- The proposed ATONE supply is around ~2/7 (28.29%) of the ATOM supply at the 
-  time of proposal 848.
+### Community Pool & reserved address
+
+The AtomOne Community Pool balance follows a different procedure, we propose
+that 1/18 of the ATONE distributed be minted to feed the Community Pool. This
+corresponds to an amount of 5,388,766 ATONE.
+
+The same amount will be minted to feed a reserved address (say
+`0x00000000000000000builddao`) for future funding of the Treasury DAO.
```
Comment (giunatale) [inline 001_ATONE_DISTRIBUTION.md:185]:
```suggestion
`0x00000000000000000buidldao`) for future funding of the Treasury DAO.
```
supposed to be "buidl"? or we changed our minds?

File: 001_ATONE_DISTRIBUTION.md [hunk 3]
```diff
@@ -164,8 +175,24 @@ ATONE distribution will be of approximately ~97 Millions.
...
+### Total supply
+
+- The proposed ATONE supply is **107,775,332 ATONE**:
+  ```
+  ATONE Distributed + Community Pool + Reserved address 
+  96,997,800        + 5,388,766      + 5,388,766        = 107,775,332
+  ```
+  which is around ~3/10 (31.43%) of the ATOM supply at the time of proposal 848
+  (342,834,268).
 - The `C` multiplier is computed as ~1,620.
```
Comment (giunatale) [inline 001_ATONE_DISTRIBUTION.md:196]:
```suggestion
- The `C` multiplier is computed as ~1.62.
```
I don't recall exactly, but this seems like a mistake, or am I wrong?

File: 001_ATONE_DISTRIBUTION.md (hunk 2 above)
Comment (tbruyelle) [reply 001_ATONE_DISTRIBUTION.md:185]: I'm not sure about this, we need to ask. In addition this isn't valid hexadecimals...

Comment (tbruyelle) [reply 001_ATONE_DISTRIBUTION.md:185]: Given last company annoucement, we can confirm it's "buidl", I commit your suggestion.

**Pull request merged**